oclint-reporters/reporters/HTMLReporter.cpp

```diff
@@ -23,13 +23,17 @@ class HTMLReporter : public Reporter
         writeHead(out);
         out << "<body>";
         out << "<h1>OCLint Report</h1>";
-        out << "<hr />";
         out << "<h2>Summary</h2>";
         writeSummaryTable(out, *results);
-        out << "<hr />";
-        out << "<table><thead><tr><th>File</th><th>Location</th>"
-            << "<th>Rule Name</th><th>Rule Category</th>"
-            << "<th>Priority</th><th>Message</th></tr></thead><tbody>";
+        out << "<h2>Details</h2>";
+        out << "<table id=\"myTable\"><thead><tr>";
```

Comment:
Perhaps `violationTable` is a better name?

Reply:
will do